tests/callbacks/test_EnqueuedEventTimer.cpp

```diff
@@ -0,0 +1,77 @@
+#include "gtest/gtest.h"
+
+#include "Kokkos_Core.hpp"
+
+#include "kokkos-utils/callbacks/EnqueuedEventTimer.hpp"
+#include "kokkos-utils/timer/Duration.hpp"
+
+/**
+ * @addtogroup unittests
+ *
+ * @c Kokkos callbacks enqueued event timer
+ * ----------------------------------------
+ *
+ * This group of tests check the behavior of @c Kokkos::utils::callbacks::EnqueuedEventTimer.
```

Comment:
```suggestion
 * This group of tests check the behavior of @c Kokkos::utils::callbacks::EnqueuedEventTimer and related.
```

(foreseeing the `WithLatency`)

Reply:
Indeed!